azure-keyvault/azure/keyvault/key_vault_client.py

```diff
@@ -67,7 +67,7 @@ def __init__(
 
 
     def create_key(
-            self, vault_base_url, key_name, kty, key_size=None, key_ops=None, key_attributes=None, tags=None, custom_headers=None, raw=False, **operation_config):
+            self, vault_base_url, key_name, kty, curve=None, key_size=None, key_ops=None, key_attributes=None, tags=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
This is breaking, and might be avoided by just putting this parameter later at the end in the parameter list in the Swagger.